CHANGES.md

```diff
@@ -1,5 +1,30 @@
 # Changes
 
+## edge-21.6.2
+
+This release fixes a problem with the HTTP body buffering that was added
+to support gRPC retries. Now, only requests with a retry configuration
+are buffered (and only when their bodies are less than 64KB).
+
+Additionally, an issue with the outbound ingress-mode proxy where forwarded
+HTTP clients could fail to detect when the target pod was deleted, causing
```

Comment:
I guess it's better, but I don't really think that _clients_ do the detection, isn't a client a browser, e.g. Chrome? I don't think it knows anything about pods.

Reply:
> isn't a client a browser, e.g. Chrome

No. An HTTP client is something that opens connections and issues requests. Browsers have clients. Ingresses have clients. Linkerd proxies have clients.

Reply:
There are obviously "clients", but which "client" is aware of target pods / deletions?

I'd be shocked if it's a generic http client. It seems more likely that it's some internal detail of how linkerd works.

That said, I'm not in a position to discuss it, and it's clear this ship has sailed.